src/lib.rs

```diff
@@ -285,6 +287,11 @@ impl WalkDir {
         self.opts.max_open = n;
         self
     }
+
+    pub fn sort(mut self, sort: bool) -> Self {
```

Comment:
Could you add documentation for this?

(I regret that I don't have `deny(missing_docs)` set, because a build with an undocumented public member should fail.)